app/views/checkpaymentchedule/CheckPaymentScheduleRows.scala

```diff
@@ -31,9 +32,11 @@ object CheckPaymentScheduleRows {
       changeLinkCall:            Call
   )(implicit lang: Language): Option[SummaryListRow] = {
       def bullets(reasons: Set[CannotPayReason]) = {
+        val htmlClass = if (reasons.size === 1) "govuk-list" else "govuk-list govuk-list--bullet"
+
         Html(
           s"""
-           |<ul class="govuk-list govuk-list--bullet">
+           |<ul class="$htmlClass">
```

Comment:
if `reason.size` is one we probably don't need the `<ul>` right?

Reply:
it might be easier to `toList` the set and pattern match to make it easier to deal with the size one case